HealthcareSystem/Backend/Model/Pharmacies/PharmacySystem.cs

```diff
@@ -29,5 +25,9 @@ public class Pharmacy
 
         [Required] 
         public bool ActionsBenefitsSubscribed { get; set; } = false;
+
+        public string grpcHost { get; set; } = "localhost";
+
+        public int grpcPort { get; set; } = 30051;
```

Comment:
Is it still necessary to keep this here for testing purposes? I'm asking since there's a PharmacySystem table in the database, won't it be read from there?

Reply:
Its for testing, it wont be hardcoded if you are referring to that.